crates/milli/src/update/new/indexer/post_processing.rs

```diff
@@ -131,6 +131,20 @@ fn compute_word_fst(
     }
 }
 
+pub fn recompute_word_fst_from_word_docids_database(index: &Index, wtxn: &mut RwTxn) -> Result<()> {
+    let fst = fst::Set::default().map_data(std::borrow::Cow::Owned)?;
+    let mut word_fst_builder = WordFstBuilder::new(&fst)?;
+    let words = index.word_docids.iter(wtxn)?.remap_data_type::<DecodeIgnore>();
```

Comment:
note for later, this could take a `&Progress` object and update it with a variable step progress type so that we can follow the rebuild of the word fst